### DISCUSSION TITLE
Can children of a horizontal container wrap to the next row if the container has a fixed width?

### DISCUSSION THREAD

**Discussion by jlsachse** [Textualize/textual#1559] [Q&A]
Hey everyone,

I am trying to archive the following behavior with CSS in Textual:
![Ap7jL](https://user-images.githubusercontent.com/45663021/212337307-23e083a8-e7ba-40d2-a412-3cbdf67ff8c2.png)

- The parent has a fixed width
- The children (tags) should shrink to their content.
- The children should wrap to the next "row" if the width of the parent is used up
- The height of the parent should grow to contain all children

I have tried to use horizontal for this, but I can't get the children to wrap to the next row. Instead they disappear to the right.

Is it possible to solve this with the current version of Textual?

**Comment by davep** [comment]
Checked with @willmcgugan and there's currently no out-of-the box way of doing this with Textual at the moment.
